migrate.js

```diff
@@ -221,6 +221,7 @@ exports.init = function init(sbot, config) {
     started = true
     debug('started')
 
+    if (sbot.db) sbot.db.setStateFeedsReady('migrating')
```

Comment:
This `'migrating'` is a trick to not let `loadStateFeeds` flag "true" for stateFeedsReady. Basically db.js might have run `loadStateFeeds` for other reasons than migration, but we want to prevent db.js from declaring "true" on stateFeedsReady, because migration will do that (see line 320 in migrate.js).

Reply:
I'm wondering if there is no concurrency issues here where migrate sets this to migrating first, and then db.js sets it to true afterwards?